resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -814,6 +814,326 @@
           label: Pharmacokinetics and pharmacodynamics/Description
           type: basic:string
           required: false
+        - name: mutation_status_kras
+          label: Mutation status/KRAS
+          type: basic:string
+          required: false
+          choices:
+            - label: KRAS G12D
+              value: kras_g12d
+            - label: KRAS G12V
+              value: kras_g12v
+            - label: KRAS G12C
+              value: kras_g12c
+            - label: KRAS G13D
+              value: kras_g13d
+            - label: KRAS Q61H
+              value: kras_q61h
+            - label: KRAS G12R
+              value: kras_g12r
+            - label: KRAS G12a
+              value: kras_g12a
+            - label: KRAS G12s
+              value: kras_g12s
+            - label: KRAS A146T
+              value: kras_a146t
+            - label: KRAS Q61L
+              value: kras_q61l
+            - label: KRAS
+              value: kras
+        - name: mutation_status_nras
+          label: Mutation status/NRAS
+          type: basic:string
+          required: false
+          choices:
+            - label: NRAS Q61R
+              value: nras_q61r
+            - label: NRAS Q61K
+              value: nras_q61k
+            - label: NRAS G13R
+              value: nras_g13r
+            - label: NRAS G12D
+              value: nras_g12d
+            - label: NRAS Q61L
+              value: nras_q61l
+            - label: NRAS
+              value: nras
+        - name: mutation_status_atm
+          label: Mutation status/ATM
+          type: basic:string
+          required: false
+          choices:
+            - label: ATM R337C
+              value: atm_r337c
+            - label: 'ATM K2811Sfs*46'
```

Comment:
Frameshifts in SnpEff are written like this `Lys2811fs`. Three letter symbols can be changed to one letter symbol. What about the rest of frameshift annotation, should this be synchronized?